### PR TITLE
Create test suite for remap_restarts.py

### PR DESCRIPTION
**Pull request by weiyuan-jiang**
This PR addresses issue #265.  It can be merged independently upon PR #238 but would not work until #238 is merged

### CODE REVIEW

File: GEOS_Util/post/tests/test_remap_restarts.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  catch  = catchANDcn(config_yaml)
+  catch.remap()
+  ana = analysis(config_yaml)
+  ana.remap()
+
+
+if __name__ == '__main__' :
+
+  # test c24_to_c12
+  base_c12 = '/gpfsm/dnb32/wjiang/pl_c24Tc12'
```
Comment (mathomp4) [inline GEOS_Util/post/tests/test_remap_restarts.py:61]: I wonder if we should put these files/etc. used for the tests in our gmao_SIteam space? Then it's at least in a "generic" directory?

Comment (weiyuan-jiang) [reply GEOS_Util/post/tests/test_remap_restarts.py:61]: Yes, eventually those hard coded path should be moved to somewhere before it is merged

**Review by sdrabenh — APPROVED** [review]

**Review by sdrabenh — APPROVED** [review]

**Review by mathomp4 — COMMENTED** [review]
Reapprove!

**Comment by mathomp4** [comment]
Not sure what's going on with approvals. I'll superpower it in.

**Pull request merged**